packages/sdk/src/sdk/middleware/addRequestSignatureMiddleware.test.ts

```diff
@@ -1,15 +1,15 @@
 import { describe, it, expect } from 'vitest'
 
-import { AppAuth, Logger } from '../services'
+import { createAppWalletClient, Logger } from '../services'
 
 import { addRequestSignatureMiddleware } from './addRequestSignatureMiddleware'
 
 describe('addRequestSignatureMiddleware', () => {
   it('generates a signature', async () => {
     const services = {
-      auth: new AppAuth(
-        '0xfe00b052EeB6DaA21e0D421d52a18DcB254d0ef7',
-        '4ac8b3eff248bfbf20b324b575c1b333d42c6db3dbe19fd587c3d1e11323a25a'
+      audiusWalletClient: createAppWalletClient(
+        '0x',
```

Comment:
Why does this work correctly with a dummy first arg?

Reply:
alternatively, let them enter _either_ api key _or_ secret, since the former can be derived